praw/models/reddit/draft.py

```diff
@@ -0,0 +1,331 @@
+"""Provide the draft class."""
+from typing import TYPE_CHECKING, Any, Dict, Optional, Union
+
+from ...const import API_PATH
+from ...exceptions import ClientException
+from .base import RedditBase
+from .subreddit import Subreddit
+from .user_subreddit import UserSubreddit
+
+if TYPE_CHECKING:  # pragma: no cover
+    import praw
+
+
+class Draft(RedditBase):
+    """A class for submissions to reddit.
+
+    **Typical Attributes**
+
+    This table describes attributes that typically belong to objects of this class.
+    Since attributes are dynamically provided (see
+    :ref:`determine-available-attributes-of-an-object`), there is not a guarantee that
+    these attributes will always be present, nor is this list necessarily complete.
+
+    ========================== ======================================================
+    Attribute                  Description
+    ========================== ======================================================
+    ``link_flair_template_id`` The link flair's ID.
+    ``link_flair_text``        The link flair's text content, or None if not flaired.
+    ``modified``               Time the submission draft was modified, represented in
+                               `Unix Time`_.
+    ``original_content``       Whether the submission draft will be set as original
+                               content.
+    ``selftext``               The submission draft's selftext. ``None`` if a link
+                               submission draft.
+    ``spoiler``                Whether the submission will be marked as a spoiler.
+    ``subreddit``              Provides an instance of :class:`.Subreddit` or
+                               :class:`.UserSubreddit` (if set).
+    ``title``                  The title of the submission draft.
+    ``url``                    The URL the submission draft links to. Will b
+    ========================== ======================================================
+
+    .. _unix time: https://en.wikipedia.org/wiki/Unix_time
+
+    """
+
+    STR_FIELD = "id"
+
+    @classmethod
+    def _prepare_data(
+        cls,
+        flair_id: Optional[str] = None,
```

Comment:
Even though this is a "private" method, I'd recommend making these mandatory keyword arguments to help establish doing so as a best practice in PRAW going forward.